src/test/resources/com/microsoft/azure/vmagent/test/jcasc/expectedAdvanced.yaml

```diff
@@ -28,7 +28,10 @@
         galleryResourceGroup: "gallery"
         gallerySubscriptionId: "e5587777-5750-4d2e-9e45-d6fbae67b8ea"
       imageTopLevelType: "advanced"
-      initScript: "yum install -y java\nyum install -y nodejs\ncat /etc/hosts"
```

Comment:
@casz I think this may block some users?

Reply:
I doubt it... we support both formats

This is just exporting.